plugins/cnfreg/proxy.go

```diff
@@ -33,17 +33,6 @@ import (
 
 // The function initializes proxy descriptor for every config model exposed by the given CNF.
 func (p *Plugin) initCnfProxy(swMod swModule) error {
-	// Register models with the local registry.
```

Comment:
I'm probably missing the idea here why we don't want the models in the local registry. Could you please explain to me the idea behind it?